Makefile

```diff
@@ -198,7 +198,7 @@ CROSSPLANE_NAMESPACE = crossplane-system
 -include build/makelib/local.xpkg.mk
 -include build/makelib/controlplane.mk
 
-UPTEST_EXAMPLE_LIST := $(shell cat cluster/test/cases.txt | grep -v '^#'  | tr '\n' ',')
+UPTEST_EXAMPLE_LIST := $(shell grep -v '^\#' cluster/test/cases.txt | paste -sd ',' -)
```

Comment:
on my mac this is not working because of 
```
Makefile:201: *** unterminated call to function `shell': missing `)'.  Stop.
```

not sure if this is now better for linux and other operating systems :D

Reply:
both yield the same for me on ubuntu